src/bin/commands/images.rs

```diff
@@ -26,6 +26,9 @@ pub struct ListImages {
     /// Container engine (such as docker or podman).
     #[clap(long)]
     pub engine: Option<String>,
+    /// output format
+    #[clap(long, default_value = "human", value_parser = clap::builder::PossibleValuesParser::new(["human", "json"]))]
+    pub output_format: String,
```

Comment:
Make this an enum, since we have a value parser and a list of enumerated values. No reason this should be a string.